.github/workflows/nightly.yml

```
@@ -14,20 +14,30 @@ jobs:
       uses: actions/checkout@master
       with:
         submodules: 'recursive'
+    - name: Download babylon.max.js
+      run: curl.exe -o Apps/node_modules/babylonjs/babylon.max.js --create-dirs https://raw.githubusercontent.com/BabylonJS/Babylon.js/master/dist/preview%20release/babylon.max.js
```

This isn't from your change, but we shouldn't be using raw github urls to get files.

```suggestion
      run: curl.exe -o Apps/node_modules/babylonjs/babylon.max.js --create-dirs https://preview.babylonjs.com/babylon.max.js
```